src/js/jsx/search/SearchBar.jsx

```diff
@@ -458,20 +482,27 @@ define(function (require, exports, module) {
             var nextHasInputValue = value.length !== 0;
             
             if (nextHasInputValue !== this.state.hasInputValue) {
-                this.setState({ hasInputValue: nextHasInputValue });
+                this.setState({
+                    hasInputValue: nextHasInputValue
+                });
             }
         },
 
         /**
          * @private
          * @type {Datalist~onKeyDown}
+         * @param {SyntheticEvent} event
+         * @param {string} selectedID the id of the selected item in the datalist
          */
         _handleKeyDown: function (event, selectedID) {
             switch (event.key) {
                 case "Return":
                 case "Enter":
-                    if (selectedID === PLACEHOLDER_ID) {
-                        this._handleDialogClick(event);
+                    // The only time selectedID is falsey is when an Adobe Stock search
```

Comment:
This comment is surely out of date since you're not hard-coding anything about Adobe Stock in this component, right?